### PR TITLE
Change $this->app->environment() !== 'production' to $this->app->isLocal()

### PR DESCRIPTION
**Pull request by Daniel-Mendes**
Change $this->app->environment() !== 'production' to $this->app->isLocal()

Just cleaner !

### CODE REVIEW

**Comment by mfn** [comment]
@Daniel-Mendes sorry but your latest change seem to have LOTS of unrelated changes!

**Comment by mfn** [comment]
@barryvdh LGTM 😏

**Review by mfn — APPROVED** [review]
@barryvdh IMHO we're good to merge here!

**Comment by ahmed-aliraqi** [comment]
@Daniel-Mendes it's not supposed to register service provider manually, because it already registered by laravel package discovery see #767

**Comment by ahmed-aliraqi** [comment]
Also the package right now doesn't support old laravel version before package discovery 😐

**Comment by mfn** [comment]
See my response in https://github.com/barryvdh/laravel-ide-helper/commit/4df96b5e62d0e7db098326ea699d88466387041e#commitcomment-42077077

TL;DR: it's fine
